receiver.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 const checksumLength = 4
+const deflateTrailerLength = 4
```

Comment:
`deflateTrailerLength` isn't being used anywhere.  Did you mean to use it below in the call to `Truncate()`?  

Also, it could be defined as a variable:

```
var deflateTrailerLength = len(deflateTrailer)

```